integration-test/src/python/local_test_runner/main.py

```diff
@@ -25,7 +25,7 @@
 ]
 # Retry variables in case the output is different from the input
 RETRY_COUNT = 5
```

Comment:
IMO if a test is failing more than 5 times it's either a bad test or our test framework is too brittle. The fact that we have to retry at all is suspect.